lib/bq_stream/bq_stream.rb

```diff
@@ -67,14 +67,16 @@ def verify_oldest_records
       log(:info, "#{Time.now}: ***** Oldest Record Revision: #{revision.attr} *****") if revision
       log(:info, "#{Time.now}: ***** Current Deploy: #{current_deploy} *****")
       return if revision && revision.attr == current_deploy
-      @bq_attributes.each do |k, v|
-        # add any records to oldest_records that are new (Or more simply make sure that that there is a record using find_by_or_create)
-        v.each do |bqa|
-          OldestRecord.find_or_create_by(table_name: k, attr: bqa)
-        end
-        # delete any records that are not in bq_attributes
-        OldestRecord.where(table_name: k).each do |rec|
-          rec.destroy unless v.include?(rec.attr.to_sym)
+      if @bq_attributes
```

Comment:
You can just do `@bq_attributes&.each do |k, v|` to avoid the extra two lines of conditional